tools/contrib/junit_xml/__init__.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
```

Comment:
When in --parallel, running the test process in the background under control by run_tests.py would require significant change to run_tests.py.  Of course simplifying this is possible and I already planned to do it.  I don't see why that improvement is required now.